bin/running_on_dart.dart

```diff
@@ -1,58 +1,48 @@
 import 'package:nyxx/nyxx.dart';
 import 'package:nyxx_commands/nyxx_commands.dart';
+import 'package:nyxx_extensions/nyxx_extensions.dart';
 import 'package:running_on_dart/running_on_dart.dart';
-import 'package:running_on_dart/src/commands/system.dart';
+import 'package:running_on_dart/src/commands/tag.dart';
 
 void main() async {
-  // Create nyxx client and nyxx_commands plugin
-  final client = NyxxFactory.createNyxxWebsocket(token, intents);
-
   final commands = CommandsPlugin(
     prefix: mentionOr((_) => prefix),
     guild: devGuildId,
-    options: CommandsOptions(logErrors: false, hideOriginalResponse: false),
+    options: CommandsOptions(logErrors: true, type: CommandType.slashOnly),
```

Comment:
Why set `prefix` if we only have slash commands?

Since no command overrides its `type` to allow text commands, we can pass `prefix: null` which will automatically set `type: CommandType.slashOnly`.